fcrepo-indexing-triplestore/src/main/java/org/fcrepo/camel/indexing/triplestore/FcrepoTripleStoreIndexerConfig.java

```diff
@@ -102,6 +108,13 @@ public String getTriplestoreBaseUrl() {
         return triplestoreBaseUrl;
     }
 
+    public String getTriplestoreAuthUsername() {
```

Comment:
Is it actually necessary to include Triplestore in the name of this method? Can it just be getAuthUsername? Same question goes for getTriplestorePassword below.

Reply:
(Same question applies to the property names, of course -- I guess these were named following the precedent of triplestoreBaseUrl, but I'm not sure that it's necessary).